iotics/lib/identity/api/high_level_api.py

```diff
@@ -31,7 +31,7 @@ def create_user_and_agent_with_auth_delegation(self, user_seed: bytes, user_key_
         :param agent_key_name: agent key name (secrets)
         :param agent_password: Optional agent password (secrets)
         :param agent_name: Optional agent name (default #agent-0) following this pattern: '#[a-zA-Z\\-\\_0-9]{1, 24}'
-        :param delegation_name: user/agent authentication delegation name
+        :param delegation_name: Optional delegation name, if None a random name will be chosen
```

Comment:
(pedantic/unimportant) Maybe (in all places where used):
_Optional delegation name. If None, a random name will be chosen._